CHANGELOG.md

```diff
@@ -1,7 +1,22 @@
-30 Aug 2019
+12th Sep 2019 (0.2.1)
```

Comment:
IMO Perhaps, reword as:

Fix bug where incorrect sensor info returned from `sensor_detail`

*"Fixes a bug where incorrect sensor info is returned from `sensor_detail` and does not match the documentation."*